addons/sale_timesheet/static/tests/mock_server/mock_models/account_analytic_line.js

```diff
@@ -0,0 +1,5 @@
+import { models } from "@web/../tests/web_test_helpers";
+
+export class AccountAnalyticLine extends models.ServerModel {
+    _name = "account.analytic.line";
```

Comment:
models should be defined in their main module (account here) otherwise we might run into some strange dependency issues if current module is not installed